src/main/java/cz/brmlab/yodaqa/io/web/WebInterface.java

```diff
@@ -47,13 +47,19 @@ public void run() {
 			@Override
 			public Object handle(Request request, Response response) {
 				String id = Integer.toString(idgen.nextInt(Integer.MAX_VALUE));
-				String text = request.queryParams("text");
+				String text=request.queryMap("question").value();
```

Comment:
@ermrk I'm confused about this line. What is the motivation to use queryMap instead of queryParams? And why are we introducing the API incompatibility here by renaming "text" to "question", what's the upside of that?